server/lib/fetch.py

```diff
@@ -258,6 +258,21 @@ def observation_existence(variables, entities):
   return result
 
 
+def entity_variables(entities):
+  """Gets the statistical variables that have observations for given entities.
+
```

Comment:
Maybe include in this comment the structure of the return type. If I understand correctly, it is a dict where key is variable dcid and value is a dict where key is entity dcid and value is a list of observations?

Reply:
Added comment. The inner level value is an empty struct. This is due to the unified observation API response.